experimental/devices/inky/lut.go

```diff
@@ -0,0 +1,222 @@
+package inky
+
+var (
+	blackLUT = []byte{
+		0x48,
```

Comment:
Can you wrap a bit to make the file a bit less long? 16 per line would be great.